docs/welcome/install.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Install ParadeDB
+---
+
+The fastest way to install ParadeDB is by pulling the ParadeDB docker image and running it locally.
+
+```bash
+docker run \
+  --name paradedb \
```

Comment:
I think we should just replace these with placeholder values and mention to the user that they should update them in production, so they can just copy-paste